ml-core-googleplay/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply from: configs.androidLibrary
+
+dependencies {
+    api project(':ml-core-base')
```

Comment:
This is the lightest tflite runtime that uses google play. It should be the preferred runtime for all SDK users with access to Google play.


Replacing `ml-core-cardscan` with `ml-core-googleplay` for cardscan example app would reduce 400kb (9% overall size), see [here](https://docs.google.com/document/d/1l4cmSbvimpLq0cZab09i4WYjYbAmRew814V1_SD9hNo/edit#bookmark=id.aydyn9sc9onz) for more details



Replacing `ml-core-default` with `ml-core-googleplay` for identity example app would reduce 1.2mb (32% overall size), see [here](https://docs.google.com/document/d/1l4cmSbvimpLq0cZab09i4WYjYbAmRew814V1_SD9hNo/edit#bookmark=id.lp5czt3hl6bb) for more details